resources/schema/spidacalc/results/component.schema

```diff
@@ -2,7 +2,7 @@
   "description": "A component is a combination of an individual item and analysis type i.e. Pole Buckling",
   "type": "object",
   "required": [
-    "id", "poles", "actual", "ratio", "allowable", "failingRatio", "status", "resultsType", "analysisType"
+    "id", "actual", "ratio", "allowable", "failingRatio", "status", "resultsType", "analysisType"
```

Comment:
wire tension components do not have "poles"

unless I am missing something, we should not have gone this long before realizing that this was invalid against our schema. Missing tests?